lib/validate.js

```diff
@@ -190,10 +190,11 @@ module.exports = {
       // In case of individual packaging we have to create a separate config for each function
       if (_.has(this.serverless, 'service.package') && this.serverless.service.package.individually) {
         this.multiCompile = true;
-        this.serializedCompile = this.configuration.serializedCompile;
         this.options.verbose &&
           this.serverless.cli.log(
-            `Using ${this.serializedCompile ? 'serialized' : 'multi'}-compile (individual packaging)`
+            `Using ${
+              this.configuration.concurrency !== Infinity ? 'concurrent' : 'multi'
```

Comment:
Shouldn't this be updated as well? Since concurrency default value is no more `Infinity`

Reply:
Before `serializedCompile` is available, `serverless-webpack` blasts everything at once in the same process, that's where the `Infinity` comes from.

Reply:
I just had a deeper look on this. When `package.individually === true` it set `multiCompile = true`, my searches tells me this property is no longer used anywhere else. In fact the original `Infinity` behavior must be explicitly forced via a `${file:...}` reference in `serverless.yml`.

Do you think it's a good idea to remove all instance of *multiCompile* and change the log to the following format instead? 
```javascript
`Individually packaging with concurrency at ${this.configuration.concurrency} entries a time.`
```

Reply:
Looks like `multiCompile` was removed here https://github.com/serverless-heaven/serverless-webpack/pull/517
So yes, good idea to change the log 👍

Reply:
All done